containers/package-builder/checkout-repositories-version.sh.j2

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+directory=$1
+repository=$2
+
+# Check if a directory exists
+if [ ! -d "$directory" ] && [[ "$repository" == "concent" ]]; then
```

Comment:
`repository` should be `/var/concent-data/concent/`, not `/concent/`. Similarly for `golem`. This way you won't have to hard-code it for `concent` and `golem`.

Reply:
Done.